### PR TITLE
Maintain whatwg.org/policies here

### PR DESCRIPTION
**Pull request by annevk**
Part of #27.

### CODE REVIEW

**Comment by annevk** [comment]
I'd prefer merging this myself so I can deal with any fallout directly. (It also requires a corresponding change to whatwg/whatwg.org.)

File: Policies.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+WHATWG participants operate under a number of policies:
+
+* [Intellectual Property Rights Policy](./IPR%20Policy.md)
+* [Workstream Policy](./Workstream%20Policy.md)
+* [Code of Conduct](https://whatwg.org/code-of-conduct)
+* [Working Mode](https://whatwg.org/working-mode)
+
+Additionally, the WHATWG Steering Group was created by the [Steering Group Agreement](./SG%20Agreement.md) and operates under the [Steering Group Policy](./SG%20Policy.md).
+
+<hr>
```
Comment (domenic) [inline Policies.md]: At some point we should fix this to match https://github.com/whatwg/whatwg.org/blob/dbcc63269a3230899384168c5047d42440f8d3d9/whatwg.org/irc#L61-L63 , either in the source files or the translation script.

Comment (annevk) [reply Policies.md]: Yeah, #78 tracks that.

**Comment by dbaron** [comment]
Looks good to me as well, assuming that the links pointing to `IPR%20Policy.md` rather than `https://whatwg.org/ipr-policy` is what's intended.

**Comment by annevk** [comment]
I had to force-push to deal with the Code of Conduct merge conflicts and link to the local copy of Code of Conduct (@dbaron the main reason that's done is so that you can browse the Markdown on GitHub if you like).

**Pull request merged**